azure-pipelines.infra-core.yaml

```diff
@@ -0,0 +1,61 @@
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: hmcts/azure-devops-templates
+      ref: refs/heads/master
+      endpoint: hmcts
+
+trigger:
```

Comment:
Do we want merged into master trigger and pr trigger?

Reply:
Yeah, going to add at a later date.